prov/tcp/src/xnet.h

```diff
@@ -98,6 +98,9 @@ typedef void  xnet_profile_t;
 #define XNET_MIN_MULTI_RECV	16384
 #define XNET_PORT_MAX_RANGE	(USHRT_MAX)
 
+/* operation flags */
+#define TCP_NO_CONNECT  (1ULL << 60)
```

Comment:
I was actually expecting a public FI_ flag similar to FI_COMPLETION etc. How does this work currently ? is it just always enabled ? We need this flag not to be set on standard fi_send/fi_recv but only on fi_read / fi_write.

Reply:
It's not a feature that will be supported beyond TCP provider so I don't think it makes much sense as a core feature.

As implemented, it can apply to the entire ep via default op_flag or to individual operations that support flags, such as fi_readmsg/fi_writemsg (and don't set as an op_flag)

Reply:
I don't understand, this is a private provider flag. How can an app be using that flag without any header defining its value ?

Reply:
You're right - I misunderstood your original point. this needs to move.